_includes/_code/container/main.dart

```diff
@@ -38,7 +38,7 @@ class _MyHomePageState extends State<MyHomePage> {
   Widget build(BuildContext context) {
     var container = new Container(
       decoration: new BoxDecoration(
-        backgroundColor: Colors.black26,
+        color: Colors.black26,
```

Comment:
For cases like this where there's a BoxDecoration with nothing but a `color`, in a Container, you can replace the `decoration` property with a `color` property that just takes the Color and avoid the BoxDecoration entirely.